images/toolchain-arm-linux-gnueabihf/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM debian:stable
```

Comment:
It seems you forgot to remove `images/toolchain-arm-linux-gnueabihf/gcc-7.4/Dockerfile`.